src/julia_internal.h

```diff
@@ -676,6 +676,9 @@ typedef union {
 #define SOURCE_MODE_NOT_REQUIRED            0x0
 #define SOURCE_MODE_ABI                     0x1
 
+#define METHOD_ISMINMAX_INVOKE_LATEST       0b0001
```

Comment:
Needs documentation on what these mean.

Reply:
I think it could use a different name as well, as IIUC it's not really related to min-max world or invokelatest. Just a name collision to avoid.